charts/yorkie-argocd/README.md

```diff
@@ -0,0 +1,93 @@
+# yorkie-argocd
+
+Installs the yorkie-argocd, which provides GitOps based continuous delivery for yorkie cluster.
+
+## Prerequisites
+
+- Kubernetes 1.24+
+- Helm 3+
+
+## Get Helm Repository Info
+
+```bash
+helm repo add yorkie https://yorkie-team.github.io/yorkie/helm-charts
+helm repo update
```

Comment:
```suggestion
$ helm repo add yorkie https://yorkie-team.github.io/yorkie/helm-charts
$ helm repo update
```

Reply:
`$` can improve readability, but it's a bit cumbersome to copy/paste commands.

For example, if I want to copy/paste CRD deletion commands, I need to:
```bash
kubectl delete crd authorizationpolicies.security.istio.io
kubectl delete crd destinationrules.networking.istio.io
kubectl delete crd envoyfilters.networking.istio.io
kubectl delete crd gateways.networking.istio.io
kubectl delete crd istiooperators.install.istio.io
kubectl delete crd peerauthentications.security.istio.io
kubectl delete crd proxyconfigs.networking.istio.io
kubectl delete crd requestauthentications.security.istio.io
kubectl delete crd serviceentries.networking.istio.io
kubectl delete crd sidecars.networking.istio.io
kubectl delete crd telemetries.telemetry.istio.io
kubectl delete crd virtualservices.networking.istio.io
kubectl delete crd wasmplugins.extensions.istio.io
kubectl delete crd workloadentries.networking.istio.io
kubectl delete crd workloadgroups.networking.istio.io
```

but putting `$` in front of all these commands will make it hard to execute commands.